cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -45,14 +45,19 @@ type Server struct {
 	// clientGetter is a field so that it can be switched in tests for
 	// a fake client. NewServer() below sets this automatically with the
 	// non-test implementation.
-	clientGetter       clientgetter.ClientGetterFunc
+	// It is meant for in-band interactions (i.e. in the context of a caller)
+	// with k8s API server
+	clientGetter clientgetter.ClientGetterFunc
+	// for out-of-band interactions (in the context of kubeapps-internal-kubeappsapis
+	// service account with k8s API server
+	backgroundClientGetter clientgetter.BackgroundClientGetterFunc
```

Comment:
It's not being used for just background/out-of-band interactions is it? Not sure, but serviceAccountClientGetter might be less misleading?

Reply:
ok, will rename it